README.md

```diff
@@ -1,17 +1,13 @@
 # GOV.UK country and territory autocomplete - what it is and how to use it
 
----
-
-**This component is not currently maintained and we are unable to respond to issues and pull requests.**
+The country and territory autocomplete is a component that helps users choose countries and territories in answers. You should only use this component for services where users have to provide their country or territory.
```

Comment:
First sentence has same wording as the [update for accessible-autocomplete](https://github.com/alphagov/accessible-autocomplete/pull/525/files):

> The [type-of-autocomplete] is a component that helps users choose countries and territories in answers.

Should we change the accessible-autocomplete wording to something like "The accessible autocomplete is a component that helps users choose answers from a list we provide"?

asdada